app/controllers/Application.scala

```diff
@@ -365,6 +355,7 @@ object Application extends Controller with Security {
   }
 
   def scheme(id: Int) = isAnalyst { identity => implicit request =>
+    // todo: update to currentSubscriberId
```

Comment:
Umm, I might want to rethink this. Right now the scheme page is where subscribers add interests, but generally its where users can learn about the scheme.
I don’t see an analyst restriction (although there *are* analyst-specific functions that might link from here)

RIchard
On May 1, 2015, at 10:05 AM, Jeremy Prevost <notifications@github.com> wrote:

> In app/controllers/Application.scala:
> 
> > @@ -365,6 +355,7 @@ object Application extends Controller with Security {
> >    }
> >  
> >    def scheme(id: Int) = isAnalyst { identity => implicit request =>
> > +    // todo: update to currentSubscriberId
> Oops, missed one. I'll add a commit to this PR.
> 
> —
> Reply to this email directly or view it on GitHub.
>